workflow.html

```diff
@@ -62,19 +65,21 @@
     </li>
     <li>
       <p>
-        Add your edits to prepare your commit by staging them. Best is to use the interactive mode (New files still
-        need to be added explictly).
+        Add your edits to prepare your commit by <a href="https://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging">staging</a> them.
```

Comment:
Having not used "git add -p" before, is that what you are describing here? If so, maybe make that more clear?

Reply:
I am sorry to intervene in a discussion I was not part of... but I think making anything dependent on some extra options on the git CLI is bound to be ignored. A lot of people (me included) use github almost exclusively through other tools (stand alone programs like Gitraken or Sourcetree, or the git(hub) extension of the Atom or VisualSourceCode editors, and I am sure there are many more). So yes, @marcoscaceres is right: any description should be made clearer and not rely on CLI options...

Reply:
At the very least, we should show both options. It can be nice to show: "here is how you can do it with Github's interface" (or whatever) - and then also complementing that with, "but you also have this option in the command line... and it gives you these really neat/advanced features if you need them".  Most group won't need much... it's sufficient if folks can send up changes, and review each other's work without making it scary or having any unnecessary  overhead.